docs/react-v9/contributing/command-cheat-sheet.md

```diff
@@ -8,6 +8,7 @@ yarn create-package # scaffolds a new package
 yarn create-component # scaffolds a new component
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31480&commitId=15482aee3c71aa0d084f8988037ae82f16f0b5f8&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|4| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|3| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>